packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -44,7 +44,7 @@ const canary = new synthetics.Canary(this, 'MyCanary', {
     code: synthetics.Code.fromAsset(path.join(__dirname, 'canary')),
     handler: 'index.handler',
   }),
-  runtime: synthetics.Runtime.SYNTHETICS_NODEJS_2_0,
+  runtime: synthetics.Runtime.SYNTHETICS_NODEJS_2_1,
```

Comment:
why did you use `SYNTHETICS_NODEJS_2_1 ` and not `SYNTHETICS_NODEJS_2_2 ` in all code samples? If the answer is "It does not matter" then it shouldn't be changed

Reply:
right. my bad , pushed 2_2 in all examples to guide customer towards latest version.